src/main/kotlin/no/nav/familie/klage/behandling/BehandlingService.kt

```diff
@@ -100,4 +104,28 @@ class BehandlingService(
             "Det eksisterer allerede en klagebehandling som ikke er ferdigstilt på fagsak med id=${fagsak.id}"
         }
     }
+
+    fun henleggBehandling(behandlingId: UUID, henlagt: HenlagtDto) {
+        val behandling = hentBehandling(behandlingId)
+
+        validerKanHenleggeBehandling(behandling)
+
+        val henlagtBehandling = behandling.copy(
```

Comment:
Vil gjerne at du legger inn historikk på ferdigstilt her :) 

Vi har ikke støtte for "henlagt" i historikk-innslag siden vi bruker steg i historikken